.github/workflows/shippable_builds.yml

```diff
@@ -674,29 +657,93 @@ jobs:
 
           cat $GITHUB_OUTPUT
 
+      - name: Rename release assets
```

Comment:
Renaming them was mainly to make sure we have sensible names in the steps such as release assets and artifact names. I don't think we should have a separate artifact to differentiate renamed vs not renamed.